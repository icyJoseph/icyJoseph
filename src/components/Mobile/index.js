import React, { Fragment } from "react";
import MainTitle from "../MainTitle";

export const Mobile = () => {
  return (
    <Fragment>
      <MainTitle title="Meet Joseph" />
      <p>Mobile View</p>;
    </Fragment>
  );
};

export default Mobile;
