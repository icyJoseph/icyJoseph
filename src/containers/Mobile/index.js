import React, { Fragment } from "react";
import MainTitle from "../../components/MainTitle";

export const Mobile = () => {
  return (
    <Fragment>
      <MainTitle title="Meet Joseph" />
      <p>Mobile View</p>
    </Fragment>
  );
};

export default Mobile;
