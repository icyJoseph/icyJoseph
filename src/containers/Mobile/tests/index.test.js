import React from "react";
import { shallow } from "enzyme";
import Mobile from "../";

describe("Mobile View", () => {
  const wrapper = shallow(<Mobile />);
  it("renders", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
