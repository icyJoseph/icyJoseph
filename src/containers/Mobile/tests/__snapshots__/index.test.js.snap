// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mobile View renders 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <MainTitle
          title="Meet Joseph"
/>,
        <p>
          Mobile View
</p>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "title": "Meet Joseph",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Mobile View",
        },
        "ref": null,
        "rendered": "Mobile View",
        "type": "p",
      },
    ],
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <MainTitle
            title="Meet Joseph"
/>,
          <p>
            Mobile View
</p>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "title": "Meet Joseph",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Mobile View",
          },
          "ref": null,
          "rendered": "Mobile View",
          "type": "p",
        },
      ],
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
